lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -77,9 +77,9 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
     HighContrastSupport.instance.addListener(_updateHighContrast);
     _addFontSizeObserver();
     _addLocaleChangedListener();
+    _addViewFocusChangeListener();
     registerHotRestartListener(dispose);
     AppLifecycleState.instance.addListener(_setAppLifecycleState);
```

Comment:
I want to refactor this to spit out a stream now... :/ :P

Reply:
> I want to refactor this to spit out a stream now... :/ :P

What would be the advante of doing something like that?

Reply:
Not passing single-purpose tear-offs around, and letting disjoint parts of the application listen to the same events. The same as why "addEventListener('click')" is better than setting "onclick" :P